tests/test_database_types.py

```diff
@@ -203,18 +313,42 @@
                             )
                         )
 
+def sanitize(name):
+    name = name.lower()
+    name = re.sub(r"[\(\)]", "", name)  #  timestamp(9) -> timestamp9
+    # Try to shorten long fields, due to length limitations in some DBs
+    name = name.replace(r"without time zone", "n_tz")
+    name = name.replace(r"with time zone", "y_tz")
+    name = name.replace(r"with local time zone", "y_tz")
+    name = name.replace(r"timestamp", "ts")
```

Comment:
I prefer shortening and never truncating names. That could lead to some nasty, nasty bugs down the line.

Reply:
That's a nice idea!